docs/how-to/deploy_charmed_magma_orchestrator.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Charmed Magma Orchestrator
+
+## Requirements
```

Comment:
Maybe we should specify that the juju controller needs to be bootstrapped with `Load-Balancer` IPs or otherwise exposed to the outside world, otherwise it will not be possible to create the relation to AGW later.

Reply:
Fixed